core/templates/dev/head/domain/question/pretest-question-backend-api.service.spec.ts

```diff
@@ -26,10 +27,12 @@ import { TranslatorProviderForTests } from 'tests/test.extras';
 
 describe('Pretest question backend API service', function() {
   var PretestQuestionBackendApiService = null;
-  var sampleDataResults = null;
+  var responseDictionaries = null;
+  var pretestQuestionObjects = null;
```

Comment:
`Question` itself is an object, so let's not add an "Objects" suffix.

Reply:
done